plugins/doc_fragments/aws.py

```diff
@@ -49,7 +49,7 @@ class ModuleDocFragment(object):
       - Passing the I(security_token) and I(profile) options at the same time has been deprecated
         and the options will be made mutually exclusive after 2022-06-01.
     type: str
-    aliases: [ aws_security_token, access_token ]
+    aliases: [ aws_session_token, session_token, aws_security_token, access_token ]
```

Comment:
I would add a note in the description section stating that these new aliases are added in a particular version. For example like this https://github.com/ansible-collections/community.vmware/blob/c571d0b97abb9ba9032a3bb6879624f00365650b/plugins/modules/vmware_host_vmnic_info.py#L88